roxie/ccd/ccdqueue.cpp

```diff
@@ -1684,6 +1684,7 @@ class CRoxieWorker : public CInterface, implements IPooledThread
                 return;
             }
 
+            CCycleTimer workerTimer;
```

Comment:
It's arguable whether the block below that creates and caches the on-demand query should be included in the timing. Either way has its pros and cons, but in reality for all the cases we are likely to care about timing, it won't arise. So either answer is acceptable.